yarn/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # CHANGELOG - yarn
 
-1.0.003-22-2017
+1.1.0/Unreleased
```

Comment:
Could you create a new section and not update this one. Since the integration has been released it would help keep a clear history.